### PR TITLE
chore: refactoring charts for initial release

### PR DESCRIPTION
**Pull request by mdebarros**
- re-factored backend dependencies/templates naming to confirm their function instead of their technology
- added supporting scripts and CI-CD temporary scripts/configs

Note: The CI-CD scripts are temporary for the time being, and will be replaced in future with something better.

### CODE REVIEW

File: package.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+if [ "$1" ]; then
+    declare -a charts=("$1")
+else
+    declare -a charts=(
+        mojaloop/example-backend
+        mojaloop/common
+        mojaloop/admin-api-svc
+        mojaloop/fspiop-transfer-api-svc
+        mojaloop/mojaloop
```
Comment (vijayg10) [inline package.sh:27]: Do we need to include the charts related to bof here?

File: update-charts-dep.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ find ./ -name "charts"| xargs rm -Rf
 find ./ -name "tmpcharts"| xargs rm -Rf
 
 declare -a charts=(
-        dependencies/backend
+        mojaloop/example-backend
         mojaloop/common
         mojaloop/admin-api-svc
         mojaloop/fspiop-transfer-api-svc
```
Comment (vijayg10) [inline update-charts-dep.sh:19]: Do we need to include the charts related to bof here?

File: package.sh (hunk 1 above)
Comment (mdebarros) [reply package.sh:27]: Yes we do.

File: update-charts-dep.sh (hunk 2 above)
Comment (mdebarros) [reply update-charts-dep.sh:19]: yes we do

File: package.sh (hunk 1 above)
Comment (vijayg10) [reply package.sh]: I added the BOF charts to the script files and pushed the commits to same PR

File: update-charts-dep.sh (hunk 2 above)
Comment (vijayg10) [reply update-charts-dep.sh]: I added the BOF charts to the script files and pushed the commits to same PR

**Review by vijayg10 — APPROVED** [review]

**Pull request merged**